README.md

```diff
@@ -75,7 +75,7 @@ We've tried to make it as easy as possible to go from fork/clone to a running in
 
 ### Credentials
```

Comment:
Not sure exactly where we should add this info, but we now have a small demo augur instance for people to use. 
Credentials here: 
{
    "connection_string": "sqlite:///:memory:",
    "database": "astros",
    "host": "chaoss.tv",
    "password": "!xpk98T6?bK",
    "port": 5432,
    "schema": "augur_data",
    "user": "eightknot",
    "user_type": "read_only"
}

Reply:
okay added credentials